web/themes/custom/droopler_subtheme/templates/views/views-view-unformatted--partner-list.html.twig

```diff
@@ -0,0 +1,31 @@
+{#
+/**
+ * @file
+ * Theme override to display a view of unformatted rows.
+ *
+ * Available variables:
+ * - title: The title of this group of rows. May be empty.
+ * - rows: A list of the view's row items.
+ *   - attributes: The row's HTML attributes.
+ *   - content: The row's content.
+ * - view: The view object.
+ * - default_row_class: A flag indicating whether default classes should be
+ *   used on rows.
+ *
+ * @see template_preprocess_views_view_unformatted()
+ */
+#}
+{% if title %}
+  <h3>{{ title }}</h3>
+{% endif %}
+{% for row in rows %}
+  {%
+  set row_classes = [
+  'col-lg-4 col-md-6 col-xs-12',
```

Comment:
A nie można po prostu tych klas dodać do ustawienia rowa w viewsie?

Reply:
można ale czy tak jest źle? jako dev wole mieć to w jednym miejscu

Reply:
w sumie, zmienie w widoku